### ISSUE TITLE
[BLOCKED] Redirect Request: Family member hub Healthcare phase 2B

### ISSUE THREAD

**Issue by kristinoletmuskat** [department-of-veterans-affairs/va.gov-team#92717]
## Statuses
**[2025-02-18] [Fran]** Comment added below asking ~~Kristin~~ (Beth Potts is new contact) if this is still pending SME review.
**[2024-10-17] [Fran]** Delayed on SME review
**[2024-09-12] [Fran]** Awaiting URL and for content to finalize H1, Targeting end of October release. Moving to Next Refinement, but can't refine until URL is provided.

| POC | Github alias(es) |
--- | --- 
| Product team point(s) of contact: | @kristinoletmuskat 

### Type of request
- [x] We are retiring or taking down a page and need to redirect the URL (complete redirect section)
- [ ] We are changing the URL of an existing page (complete redirect section)
- [ ] We need a custom vanity URL (complete vanity URL section)
- [ ] We are removing a temporary redirect  (add relevant background and link to original redirect request)

### Implementation date
When does this request need to be live? **NOTE:** Redirects in vsp-platform-revproxy auto-deploy M-Th. Friday launches should be avoided.
Choose one:
- [x] On a specific date, connected to a program or publishing requirement. **DATE: late October TBD**
- [ ] Any day prior to a deadline/latest possible date. **DEADLINE:** 
- [ ] Any day after an earliest possible date. **EARLIEST DATE:**
- [ ] During a specific date range: ***EARLIEST DATE TO LATEST DATE:***
- [ ] No hard timing requirements, publish when possible.

### Redirects 
| Current URL                                                                          | Redirect to                                                             | Status                             |
|----------------------------------------------------------------------------------|-------------------------------------------------------------------------|------------------------------------|
| https://www.va.gov/COMMUNITYCARE/programs/dependents/CLFMP.asp                   | Camp Lejeune page (URL TBD)                                             | Waiting for content to finalize H1 |

### Process, Roles and Responsibilities
- [ ] Requesting team: Above information is provided
- [ ] Requesting team: All appropriate VA stakeholders are notified as appropriate
- [ ] IA: Request is vetted and documented and implementation plan is clear 
- [ ] IA: Request is assigned to appropriate team for implementation 
- [ ] Implementation team: Work is complete
- [ ] Implementation team: Validated in production
- [ ] Requesting team: Validates everything is correct in production and closes ticket

**Comment by FranECross** [comment]
@kristinoletmuskat Just checking back on this item. Thanks!

**Comment by kristinoletmuskat** [comment]
Hello sorry for the delay @FranECross! I think we are another 3 weeks out on this - SME review is taking longer than we expected.

**Comment by FranECross** [comment]
@kristinoletmuskat Just wanted to confirm that this is still blocked on SME review (don't want it to fall between the cracks). Thanks!

**Comment by kristinoletmuskat** [comment]
Hey @FranECross just a reminder that I am not on CAIA anymore! @bethpotts should be able to help you with this one.

**Comment by VCM86** [comment]
Closing this ticket and will create a new ticket when we're ready to do the redirect.
